packages/interactor/package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "lint": "eslint '{src,test}/**/*.ts'",
     "mocha": "mocha -r ts-node/register",
-    "test:unit": "mocha -r ts-node/register test/**/*.test.ts",
+    "test:unit": "mocha -r ts-node/register 'test/**/*.test.ts'",
```

Comment:
I think single quotes might be a no-no in windows shell. /cc @jbolda

Reply:
Aye, the shell processes the glob first with single quotes on windows. Escaped double quotes is pretty much always going to be the preferred option here.